kolibri_exercise_perseus_plugin/assets/src/views/PerseusRendererIndex.vue

```diff
@@ -449,14 +447,22 @@
               return Promise.reject(`item data for ${this.itemId} not found`);
             })
             .then(itemResponse => {
-              const graphieImages = Object.keys(Array.from(itemResponse.matchAll(graphieRegex)).reduce((acc, value) => {
-                acc[value[1]] = true;
-                return acc;
-              }, {}));
-              const images = Object.keys(Array.from(itemResponse.matchAll(imageRegex)).reduce((acc, value) => {
-                acc[value[1]] = true;
-                return acc;
-              }, {}));
+              const graphieMatches = {};
+              const imageMatches = {};
+              const matches = Array.from(itemResponse.matchAll(allImageRegex));
```

Comment:
Are you able to to test this on IE11? caniuse.com says `matchAll` [isn't a supported string method on IE11](https://caniuse.com/?search=matchall)

Reply:
For posterity, it seems to be polyfilled!

![Screenshot from 2021-04-19 11-04-51](https://user-images.githubusercontent.com/1680573/115282685-564e2a80-a0ff-11eb-9f3a-80702fb41a8e.png)